library/Claim.class.php

```diff
@@ -1079,6 +1080,12 @@ function box14qualifier()
       return empty($this->billing_options['box_14_date_qual']) ? '431' :
               $this->billing_options['box_14_date_qual'];
   }
+
+  function en_pos_code()
+  {
```

Comment:
Per the request made, I have removed the business logic out of the gen_x12 and removed the other POS function since it is not longer needed and consolidated into the facilityPOS function which maintains the continuity of the cascading functionality.